service/worker.go

```diff
@@ -427,8 +427,13 @@ func (s *WorkerService) initiateTxSignWithVerifier(ctx context.Context, signRequ
 		return err
 	}
 
+	verifierHost := s.cfg.Server.VerifierHost
+	if verifierHost == "" {
+		verifierHost = "localhost" // Default to localhost if not specified
+	}
+
 	signResp, err := http.Post(
-		fmt.Sprintf("http://localhost:%d/signFromPlugin", s.verifierPort),
+		fmt.Sprintf("http://%s:%d/signFromPlugin", verifierHost, s.verifierPort),
```

Comment:
This should either expect the port to be in the provided configuration value, or have a separate verifierPort in the config to maintains the `%s:%d` format - right now, this will fail for most deployments since verifier port won't be 8080/8081, but probably just 443 behind cloudflare.